KarhooUISDK/FeatureFlags/KarhooFeatureFlagProvider.swift

```diff
@@ -9,7 +9,8 @@
 import Foundation
 
 protocol FeatureFlagProvider {
-    func get() -> FeatureFlagsModel?
+    func getRemoteFlags() -> FeatureFlagsModel?
```

Comment:
I am not really sure if the interface should care if a feature flag is remote or not. Couldn't we create a simpler api for that? Maybe return the boolean directly?

Reply:
In my opinion it should work like this. And then, for every single case in app you will decide how SDK should act in case of missing feature flag. If provider is not able to get remote values receiver should know about it. Mayby, for some cases we will have 3 scenarion: one for true, one for false and third path in case of missing flag.

Reply:
In my opinion it should work like this. And then, for every single case in app you will decide how SDK should act in case of missing feature flag. If provider is not able to get remote values receiver should know about it. Mayby, for some cases we will have 3 scenarion: one for true, one for false and third path in case of missing flag.